common-util/abiAndAddresses/tokenomics.js

```diff
@@ -1,9 +1,10 @@
 export const TOKENOMICS = {
-  contractName: 'TokenomicsProxy',
+  contractName: 'Tokenomics',
```

Comment:
doesn't affect anything, but the name has not changed https://etherscan.io/address/0xc096362fa6f4A4B1a9ea68b1043416f3381ce300#code